core/http-auth-spi/src/main/java/software/amazon/awssdk/http/auth/spi/internal/DefaultSyncSignRequest.java

```diff
@@ -52,6 +57,13 @@ public BuilderImpl(IdentityT identity) {
             super(identity);
         }
 
+        private BuilderImpl(DefaultSyncSignRequest<IdentityT> request) {
```

Comment:
The other builder methods are causing the spotbugs check to fail because they are public:
```
software.amazon.awssdk.http.auth.spi.internal.DefaultSyncSignRequest$BuilderImpl()V must not be public.

software.amazon.awssdk.http.auth.spi.internal.DefaultSyncSignRequest$BuilderImpl(Lsoftware/amazon/awssdk/identity/spi/Identity;)V must not be public.

[ERROR] Medium: Bad toBuilder implementation. See the SpotBugs logs for problem details. [software.amazon.awssdk.http.auth.spi.internal.DefaultSyncSignRequest$BuilderImpl] At DefaultSyncSignRequest.java:[lines 52-53] BAD_TO_BUILDER

[ERROR] Medium: Bad toBuilder implementation. See the SpotBugs logs for problem details. [software.amazon.awssdk.http.auth.spi.internal.DefaultSyncSignRequest$BuilderImpl] At DefaultSyncSignRequest.java:[lines 57-58] BAD_TO_BUILDER


```

Reply:
fixed.